sr/comp/mixtape/mixtape.py

```diff
@@ -130,6 +146,10 @@ def generate_play_actions(
             elif 'magicq_playback' in track:
                 action, name = self.get_run_cue_action(track, current_offset)
             elif 'obs_video' in track:
+                load_action, preroll_time = self.get_load_video_action(track, current_offset)
+                # priority 1 used since timing of load not critical
+                # and we don't want to delay other actions
+                yield track['start'] - preroll_time, 1, load_action
```

Comment:
Would be great to have an equivalent `print` for this scheduled item too.

Reply:
bd6430ae7872f7194a918a4cb763801e9b4faa5f